src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -355,11 +356,17 @@ export default {
         return 0;
       });
     },
-    focusReference(index) {
-      this.$refs.match[index].$el.focus();
+    focusReference() {
+      if (!this.$refs.match || this.$refs.match[this.focusedIndex].$el) return false;
+      return this.$refs.match[this.focusedIndex].$el.focus();
+    },
+    handleDownKeyInput() {
+      this.focusedIndex = 0;
+      this.focusReference();
     },
-    scrollIntoView(index) {
-      this.$refs.match[index].$el.scrollIntoView({
+    scrollIntoView() {
+      if (!this.$refs.match || this.$refs.match[this.focusedIndex]) return false;
+      return this.$refs.match[this.focusedIndex].scrollIntoView({
```

Comment:
Why sometimes `this.$refs.match[this.focusedIndex]` and others `this.$refs.match[this.focusedIndex].$el` ?

I feel like you could create a computed for this 🤔

Reply:
Fixed at https://github.com/apple/swift-docc-render/pull/736/commits/8d8c17e136086ff633c105f2f4aa570e54361551